cookbooks/bcpc-hadoop/recipes/jmxtrans_agent_datanode.rb

```diff
@@ -0,0 +1,14 @@
+template node['bcpc']['hadoop']['jmxtrans_agent']['datanode']['xml'] do
+  source 'jmxtrans_agent_datanode.xml.erb'
+  mode 0644
+  variables ( lazy {
```

Comment:
Why the lazy load? I don't see any of these node attributes being edited in converge time.